core/node/eth_sender/src/eth_tx_manager.rs

```diff
@@ -542,6 +542,7 @@ impl EthTxManager {
         let pool = self.pool.clone();
 
         loop {
+            tokio::time::sleep(self.config.tx_poll_period()).await;
```

Comment:
why is this needed?

Reply:
just a small refactoring, for simpler code with continue